pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -186,6 +189,16 @@ func (c *Cluster) completeOSDsForAllNodes(config *provisionConfig, configOSDs bo
 					logger.Infof("orchestration status config map result channel closed, will restart watch.")
 					w.Stop()
 					<-time.After(100 * time.Millisecond)
```

Comment:
i think we can increase this 100ms wait.  if the configmap list operation 2 lines down fails, we'll just be in a tight loop resetting the watch over and over 10 times per second.